docker/build.sh

```diff
@@ -44,6 +44,6 @@ done
 
 dotnet msbuild Datadog.Trace.proj -t:RestoreAndBuildSamplesForPackageVersions -p:Configuration=$buildConfiguration -p:ManagedProfilerOutputDirectory="$PUBLISH_OUTPUT" -p:TargetFramework=$publishTargetFramework
 
-for proj in Datadog.Trace.ClrProfiler.IntegrationTests ; do
+for proj in Datadog.Trace.IntegrationTests Datadog.Trace.OpenTracing.IntegrationTests Datadog.Trace.ClrProfiler.IntegrationTests ; do
```

Comment:
I just noticed this as well while working on #611. We don't [run the other projects in CI](https://github.com/DataDog/dd-trace-dotnet/blob/5e6adfb739de4a55b9f2f2efa3380b6be28072af/.azure-pipelines/integration-tests.yml#L277) either.

Reply:
I didn't realize that. I'll address that in this PR

Reply:
Fixed in fdd7f513704e2e2ed71e6f4f3b206c36a3d527ff